redbot/cogs/trivia/data/lists/mlb.yaml

```diff
@@ -0,0 +1,207 @@
+- Ferugson Jenkins
+- ferugson jenkins
+What was specifically designed to be the size and weight of a baseball? Walkie talkie or Grenade:
+- Grenade
+- grenade
+What MLB pitcher would go on to become a U.S. Senator? Jim Bunning or Roger Clemens:
+- Jim Bunning
+- jim bunning
+Who was the shortest man ever to play Major League Baseball? Alex Armstri, Fred Portrok, or Eddie Gaedal:
+- Eddie Gaedal
+- eddie gaedal
+Who was the first U.S. president to throw the ceremonial first pitch on Opening Day? Grover Cleveland or William Taft:
+- William Taft
+- william taft
+What is the most popular ballpark food item? Nachos or Hot dogs:
+- Hot dogs
+- hot dogs
+What state has produced the fewest MLB players? Idaho, Hawaii, or Alaska:
+- Alaska
+- alaska
+What was Babe Ruth's first name? George, Babe, Henry:
+ - George
+ - george 
+What state has produced the most MLB players? New York, Texas, California:
+- Cali
+- California
+- california
+- cali 
+What team became the Washington Nationals in 2005? Washington Senators, Monteral Expos:
+- Monteral Expos
+- monteral expos
+- Expos
+- expos
+What team won the 2016 World Series? Cubs, Royals, or Indians:
+- Cubs
+- cubs
+How old was the youngest MLB player? 13, 17, or 15:
+- 15
+- fifteen
+What MLB team did former Denver Broncos QB Tim Tebow sign with in 2016? Mets, Reds, or Rockies:
+- Mets
+- mets
+What is the most expensive baseball card? Mickey Mantle, Honus Wagner, or Babe Ruth:
+- Honus Wagner
+- honus wagner
+- Honus wagner
+How many rookies did the 2005 Atlanta Braves have on their roster? 11, 21, 18:
+- 18
+- eighteen
+What was the name of Shoeless Joe Jackson's favorite bat? Black Betsy, War Hammer, Wonder Boy:
+- Black Betsy
+- Black betsy
+- black betsy
+What color are the "mile high" seats at Coors Field? Green, Purple, Silver:
+- Purple
+- purple
+How many of the Miami Marlins' starting pitchers had winning records in 2019? 4,6, 0:
+- 0
+- zero
+- Zero
+Who is the only player of the modern era to lead the league in home runs in a season without hitting a ball over the fence? Ty Cobb, Sam Crawford, or Rudy York:
+- Ty Cobb
+- ty cobb
+- Ty cobb
+Who was the first Puerto Rican player to receive a World Series MVP Award? Carlos Beltre, Roberto Clemente, or Ivan Rodriguez:
+- Roberto Clemente
+- Roberto clemente
+- roberto clemente
+What year did the first recorded baseball game take place? 1699, 1843, or 1749:
+- 1749
+What pitcher holds the season record for lowest ERA? Tim Keefe, Bob Gibson, Dutch Lenort:
+- Tim Keefe
+- Tim keefe
+- tim keefe
+What is the most stolen base? 3rd, 2nd, or Home:
+- 2nd
+- 2nd Base
+- 2nd base
+- Second base
+What is the name of the Tampa Bay Rays AAA minor league team? Knights, Bulls, or Jumbo Shrimps:
+- Bulls
+- bulls
+Who was the first American League player to hit a home run in his first at bat? Luke Stuart, Elmar Smaf, Jo Mulligan:
+- Luke Stuart
+- Luke stuart
+- luke stuart
```

Comment:
<https://en.wikipedia.org/wiki/List_of_Major_League_Baseball_players_with_a_home_run_in_their_first_major_league_at_bat>
This list has some different data

Reply:
I completely changed the question as it was not a very good trivia question.